bot_management/bot_body.py

```diff
@@ -0,0 +1,126 @@
+                    return e
+                except BinanceOrderException as e:
+                    # error handling goes here
+                    return e
+            else:
+                try:
+                    buy_order = binance_client.create_order(
+                        symbol='BTCUSDT',
+                        side=SIDE_BUY,
+                        type=ORDER_TYPE_MARKET,
+                        quantity=amount)
+                    client.write_points(json_body)
+                except BinanceAPIException as e:
+                    # error handling goes here
+                    print(e)
+                    return 'insuficeint_funds'
+                except BinanceOrderException as e:
+                    # error handling goes here
+                    print(e)
+                    return 'command is writen wrong'
+
+        elif what_to_do == "sell":
+            if test == 3:
+                # only needed for tests
+                try:
+                    buy_order = binance_client.create_test_order(
+                        symbol='LTCUSDT',
+                        side=SIDE_SELL,
+                        type=ORDER_TYPE_MARKET,
+                        quantity=amount)
+                    client.write_points(json_body)
+                except BinanceAPIException as e:
+                    # error handling goes here
+                    return e
+                except BinanceOrderException as e:
+                    # error handling goes here
+                    return e
+            else:
+                try:
+                    buy_order = binance_client.create_order(
+                        symbol='BTCUSDT',
+                        side=SIDE_SELL,
+                        type=ORDER_TYPE_MARKET,
+                        quantity=amount)
+                    client.write_points(json_body)
+                except BinanceAPIException as e:
+                    # error handling goes here
+                    return e
+                except BinanceOrderException as e:
+                    # error handling goes here
+                    return e
+
+        else:
+            client.write_points(json_body)
+            print("waiting")
+
+    def start_bot(self, api_key, api_secret, test):
+        """bot function that loops it's work till told otherwise(not implemented yet)"""
+        if test == 1:
+            # only needed for tests
+            sleep(5)
+        elif test == 2:
+            # only needed for tests
+            binance_client = Client(api_key, api_secret)
+            Bot.switcher(self, self.bot_id, binance_client, 1, "invest", test)
+        else:
+            binance_client = Client(api_key, api_secret)
+            while self.running:
+                print(self.bot_id)
+                amount_from_AI = 1
+                command_from_AI = "invest"
+                Bot.switcher(self, self.bot_id, binance_client, amount_from_AI, command_from_AI)
```

Comment:
Now I've just realised that we use a `switcher` name. I cannot instatly tell what this method does. Maybe propose some better naming convention?